src/modules/removeTask.js

```diff
@@ -0,0 +1,11 @@
+import DataClass from './dataClass.js';
+import show from './show.js';
+// Method to add a new task to the array tasks
+const removeTask = (i) => {
+  // Add the task in the storage
+  DataClass.removeTaskData(i);
+  // Refresh the bookUl in the UI
+  show();
+};
```

Comment:
![TodoList (copy)](https://user-images.githubusercontent.com/93039359/179320729-37cb6117-a38c-48f8-a377-3ce77c29e7f1.png)

- [ ] It is great that you have implemented the button for deleting the todo :+1: but it only works when you refresh the page after adding a todo. Please fix it. :blush:

- [ ] Your delete button only works once after refreshing the page. But if we wanna remove multiple todos then we have to refresh the page after removing single todo. Please fix it :+1: